src/member-api.js

```diff
@@ -162,22 +167,17 @@ export class MemberApi extends TypedEmitter {
           throw new ExhaustivenessError(inviteResponse.decision)
       }
     } finally {
-      this.#deviceIdsWithPendingInvites.delete(deviceId)
+      this.#outboundInvitesByDevice.delete(deviceId)
     }
   }
 
   /**
-   * Send an invite and return the response. Will throw if no response is
-   * received within the timeout.
-   *
    * @param {string} deviceId
    * @param {Invite} invite
-   * @param {object} options
-   * @param {number} options.timeout
+   * @param {AbortSignal} signal
    */
-  async #sendInviteAndGetResponse(deviceId, invite, { timeout }) {
+  async #sendInviteAndGetResponse(deviceId, invite, signal) {
```

Comment:
maybe check if signal is already aborted before continuing?

Reply:
Done in c38b70823f1540dfce29e7276c4bdcc7b646ff04.